gnd/src/test/java/com/google/android/gnd/HiltTestWithParameterizedRobolectricRunner.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd;
+
+import org.junit.runner.RunWith;
+import org.robolectric.ParameterizedRobolectricTestRunner;
+import org.robolectric.RobolectricTestRunner;
+
+/**
+ * Injects Hilt dependencies during setUp and uses {@link RobolectricTestRunner} for
+ * instrumentation.
+ */
+@RunWith(ParameterizedRobolectricTestRunner.class)
+public abstract class HiltTestWithParameterizedRobolectricRunner extends BaseHiltTest {}
```

Comment:
I'm not sure if it's worth creating a new class to reuse one annotation. It's probably easier to read if we just apply the annotation at the concrete class level instead.

Reply:
Done